### PR TITLE
[WAIT FOR DEPLOY] Admin redirections to kitt

### PR DESCRIPTION
**Pull request by krokrob**
this adds 302 to kitt for city and camp admin
@ssaunier please review
Environment variable `KITT_BASE_URL` needed
DO NOT MERGE before kitt branch `drop-alumni` AND  www branch `kitt-graphql-client` deployed

### CODE REVIEW

**Comment by krokrob** [comment]
DO NOT MERGE before kitt branch `drop-alumni` deployed

File: app/controllers/batches_controller.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def show
   end
 
   def new
+    redirect_to "#{ENV['KITT_BASE_URL']}/admin/cities/#{@city.slug}/camps/new"
```
Comment (ssaunier) [inline app/controllers/batches_controller.rb]: @krokrob Did this URL change?

File: app/controllers/batches_controller.rb [hunk 2]
```diff
@@ -33,6 +34,11 @@ def create
   end
 
   def edit
+    if @batch.slug.blank?
+      redirect_to "#{ENV['KITT_BASE_URL']}/admin/camps"
```
Comment (ssaunier) [inline app/controllers/batches_controller.rb]: Same here

File: app/controllers/batches_controller.rb [hunk 3]
```diff
@@ -33,6 +34,11 @@ def create
...
+    else
+      redirect_to "#{ENV['KITT_BASE_URL']}/admin/camps/#{@camp.slug}/edit"
```
Comment (ssaunier) [inline app/controllers/batches_controller.rb]: Same here

File: app/controllers/cities_controller.rb [hunk 4]
```diff
@@ -12,10 +13,11 @@ def index
   end
 
   def show
-    @managers = User.joins(:cities).where(cities: { slug: @city.slug})
+    redirect_to "#{ENV['KITT_BASE_URL']}/cities/#{@city.slug}/edit"
```
Comment (ssaunier) [inline app/controllers/cities_controller.rb]: Same here

File: app/controllers/cities_controller.rb [hunk 5]
```diff
@@ -12,10 +13,11 @@ def index
...
   end
 
   def edit
+    redirect_to "#{ENV['KITT_BASE_URL']}/cities/#{@city.slug}/edit"
```
Comment (ssaunier) [inline app/controllers/cities_controller.rb]: Same here

**Comment by krokrob** [comment]
@ssaunier done

**Pull request merged**